.gitignore

```diff
@@ -21,6 +21,7 @@ nv-settings.lua
 lv-settings.lua
 lua/lv-user/
 lua/lv-user-config/
+tests/artifacts/
```

Comment:
```suggestion
```
This is now fixed

Reply:
Ah, tests no longer produce these locally?